app/controllers/v0/profile/connected_applications_controller.rb

```diff
@@ -49,19 +47,20 @@ def apps_from_grants
         payload = { icn: }
         url_with_params = "#{grant_url}?#{URI.encode_www_form(payload)}"
         headers = { apiKey: Settings.connected_apps_api.connected_apps.api_key }
-        response = RestClient::Request.execute(method: :get, url: url_with_params, headers:)
 
-        if response.code == 200
+        response = Faraday.get(url_with_params, {}, headers)
+
+        if response.status == 200
           parsed_response = JSON.parse(response.body)
           lhapps = parsed_response['apps']
           lhapps.each do |lh_app|
             app = build_apps_from_data(lh_app)
-            (data ||= []) << app
+            data << app
           end
-          { 'data' => data }
         end
-      rescue
-        { data: [] }
+        data
+      rescue Faraday::Error
```

Comment:
is there potential for any other errors being thrown here? parsing json, the `build_apps_from_data` method, retrieving a setting, etc? If so, is it problematic to only rescue Faraday errors?

Also, what would happen if it iterates over a couple of the `lhapps` before throwing an error, at that point, there could be something in `data`, right? so it wouldn't return the empty array that it did before? I'm not sure if this is possible or if it's a problem. Just thinking out loud. But I think just returning `[]` would at least provide the same behavior